trio/socket.py

```diff
@@ -6,6 +6,10 @@
 # here.
 # We still have some underscore names though but only a few.
 
+
+# Uses `from x import y as y` for compatibility with `pyright --verifytypes` (#2625)
+# It ignores CONSTANTS From socket though.
```

Comment:
Hmm what does this mean? Also `From` -> `from`.

Reply:
see discussion in previous conversation.

fixed~